scripts/coverage.sh

```diff
@@ -16,20 +16,17 @@ rm -rf "$COVERAGE_DIR" "$COVERAGE_FILE"
 mkdir "$COVERAGE_DIR"
 chmod +x "$PROJECT_DIR"/bin/dredd
 
-# Testing & coverage
-istanbul cover ./node_modules/.bin/_mocha -- \
-    "test/**/*-test.coffee" \
-    --compilers=coffee:coffee-script/register \
-    --recursive
 
 export COVERAGE_DIR
-cd ..
+
+
+# Testing & coverage
+istanbul cover ./node_modules/.bin/_mocha -- \
```

Comment:
I'm missing a comment mentioning the GitHub issue about /bin/dredd coverage being ignored.